### DISCUSSION TITLE
Long diagnostic messages are off screen

### DISCUSSION THREAD

**Discussion by idelice** [LazyVim/LazyVim#2070] [General]
<img width="1500" alt="Screenshot 2023-11-26 at 22 23 20" src="https://github.com/LazyVim/LazyVim/assets/59118341/f6e6ad6f-74e0-42c7-bc8c-e9a9a4ccea87">

I can't read the whole message and I don't know how to.

What's the approach here to view the full message?

**Comment by dpetka2001** [comment]
You can use `<leader>cd` with the cursor on the line to show you the line diagnostics.
